### ISSUE TITLE
Rust type parameters with lifetime are colored incorrectly

### ISSUE THREAD

**Issue by jkcclemens** [akamud/vscode-theme-onedark#26]
Code:
```rust
fn whatever(&self, closure: Box<Fn() + 'static>) {
  // do nothing
}
```

Atom:
![Atom](http://i.imgur.com/Nl7f10i.png)

VS Code:
![VS Code](http://i.imgur.com/HZEbvRR.png)

The `Fn()` part should not be colored as a lifetime.

**Comment by akamud** [comment]
I'll fix it and release a new version soon.

**Comment by akamud** [comment]
This is fixed in version 0.3.

Thank for the patience while I fixed this :)
